src/InvestJob.jsx

```diff
@@ -187,18 +184,42 @@ export default class InvestJob extends React.Component {
       '--headless',
       `-d ${datastackPath}`,
     ];
-    const investRun = spawn(path.basename(investExe), cmdArgs, {
-      env: { PATH: path.dirname(investExe) },
-      shell: true, // without true, IOError when datastack.py loads json
-    });
+    if (process.platform !== 'win32') {
+      this.investRun = spawn(path.basename(investExe), cmdArgs, {
+        env: { PATH: path.dirname(investExe) },
+        shell: true, // without shell, IOError when datastack.py loads json
+        detached: true, // counter-intuitive, but w/ true: invest terminates when this shell terminates
+      });
+      this.investRun.terminate = () => {
+        if (this.state.jobStatus === 'running') {
+          // the '-' prefix on pid sends signal to children as well
+          process.kill(-this.investRun.pid, 'SIGTERM');
+        }
+      };
+    } else {  // windows
+      this.investRun = spawn(path.basename(investExe), cmdArgs, {
+        env: { PATH: path.dirname(investExe) },
+        shell: true,
+      });
+      this.investRun.terminate = () => {
+        if (this.state.jobStatus === 'running') {
+          // process.kill(-this.investRun.pid, 'SIGTERM'); // does not kill
+          // this.investRun.kill(); // does not kill
+          // This kills, but no chance to handle it and show 'Run Cancelled'
+          exec(`taskkill /pid ${this.investRun.pid} /t /f`);
+          // exec(`taskkill /pid ${this.investRun.pid} /t`) // does not kill
+        }
+      };

```

Comment:
I did a little bit of looking into this and I probably found what you did; a lot of nodejs issues about how `process.kill` doesn't function correctly on Windows. It also seems like Windows doesn't use Signals in the same way.

The threads I was on: 
- https://github.com/nodejs/node/issues/3617
- https://github.com/nodejs/node/issues/12378